provider/cluster/service.go

```diff
@@ -232,6 +234,7 @@ loop:
 		if manager != nil {
 			manager := <-s.managerch
 			s.log.Debug("manager done", "lease", manager.lease)
+			// TODO: ??? shutdown each manager?
```

Comment:
this is "drain mode" - it waits for all of the resources it created to complete.

the managers are signaled to shutdown automatically by [this](https://github.com/ovrclk/akash/blob/e126a9d1838a8b369221188e97b5b895239d87fe/provider/cluster/manager.go#L64)